bin/generate-readme.mjs

```diff
@@ -152,13 +152,23 @@ async function processMetadata (repoMetadata) {
         supportDocsUrl = 'https://cloud.google.com/stackdriver/docs/getting-support';
       }
 
+      if (!supportDocsUrl.match(/https/)) {
+        supportDocsUrl = `https://${supportDocsUrl}`
+      }
+      
+      let res;
+      let remoteUrlExists = true;
       // if URL doesn't exist, fall back to the generic docs page
-      const res = await request({
-        url: supportDocsUrl,
-        method: 'HEAD',
-        validateStatus: () => true
-      });
-      const remoteUrlExists = res.status !== 404;
+      try {
+        res = await request({
+          url: supportDocsUrl,
+          method: 'HEAD',
+          validateStatus: () => true
+        });
+      } catch (err) {
```

Comment:
I would look at the error `code` and only set `remoteUrlExists = false` if the error returned is a 404 (indicating that it doesn't exist).

This way we don't mess up the README.md if there happen to be spurious 500s.